### DISCUSSION TITLE
Bitmap Image scrollable

### DISCUSSION THREAD

**Discussion by vo1s** [AvaloniaUI/Avalonia#14613] [Q&A]
I have a Bitmap Image in A4 format size and I want to display it it in full size but make it scrollable. So for example an area half of the image size and to scroll that image inside of it. What can I do to reach this goal?

**Comment by timunie** [comment]
wrap it inside a ScrollViwer. Also worth to check out PanAndZoomControl.

**Comment by sn4k3** [comment]
If you want a dedicated control with advanced features for Bitmap check out [UVtools.AvaloniaControls - AdvancedImageBox](https://www.nuget.org/packages/UVtools.AvaloniaControls), it allow you to set zoom (In your case 100%) while also allow pan and scroll
